examples/model_selection/plot_roc_crossval.py

```diff
@@ -3,77 +3,86 @@
 
-Example of Receiver Operating Characteristic (ROC) metric to evaluate
-classifier output quality using cross-validation.
+This example presents how to estimate and visualize the variance of the Receiver
+Operating Characteristic (ROC) metric using cross-validation.
 
-ROC curves typically feature true positive rate on the Y axis, and false
-positive rate on the X axis. This means that the top left corner of the plot is
-the "ideal" point - a false positive rate of zero, and a true positive rate of
-one. This is not very realistic, but it does mean that a larger area under the
-curve (AUC) is usually better.
-
-The "steepness" of ROC curves is also important, since it is ideal to maximize
-the true positive rate while minimizing the false positive rate.
+ROC curves typically feature true positive rate (TPR) on the Y axis, and false
+positive rate (FPR) on the X axis. This means that the top left corner of the
+plot is the "ideal" point - a FPR of zero, and a TPR of one. This is not very
+realistic, but it does mean that a larger area under the curve (AUC) is usually
+better. The "steepness" of ROC curves is also important, since it is ideal to
+maximize the TPR while minimizing the FPR.
 
 This example shows the ROC response of different datasets, created from K-fold
 cross-validation. Taking all of these curves, it is possible to calculate the
-mean area under curve, and see the variance of the curve when the
+mean Area Under the Curve (AUC), and see the variance of the curve when the
 training set is split into different subsets. This roughly shows how the
-classifier output is affected by changes in the training data, and how
-different the splits generated by K-fold cross-validation are from one another.
+classifier output is affected by changes in the training data, and how different
+the splits generated by K-fold cross-validation are from one another.
 
 .. note::
 
-    See also :func:`sklearn.metrics.roc_auc_score`,
-             :func:`sklearn.model_selection.cross_val_score`,
-             :ref:`sphx_glr_auto_examples_model_selection_plot_roc.py`,
-
+    See :ref:`sphx_glr_auto_examples_model_selection_plot_roc.py` for a
+    complement of the present example explaining the averaging strategies to
+    generalize the metrics for multiclass classifiers.
 """
 
 # %%
-# Data IO and generation
-# ----------------------
-import numpy as np
+# Load and prepare data
+# =====================
+#
+# We import the :ref:`iris_dataset` which contains 3 classes, each one
+# corresponding to a type of iris plant. One class is linearly separable from
+# the other 2; the latter are **not** linearly separable from each other.
+#
+# In the following we binarize the dataset by dropping the "virginica" class
+# (`class_id=2`). This means that the "versicolor" class (`class_id=1`) is
+# regarded as the positive class and "setosa" as the negative class
+# (`class_id=0`). We also add noisy features to make the problem harder.
```

Comment:
We can first load the data and add a new marker `# %%` and the sentence to say that we add the noise.

```suggestion
# (`class_id=0`).
```